ledger/sandbox-common/src/main/scala/platform/sandbox/banner/Banner.scala

```diff
@@ -8,16 +8,18 @@ import java.io.PrintStream
 import scala.io.Source
 
 object Banner {
-  def show(out: PrintStream): Unit = {
-    val resourceName = "banner.txt"
-    if (getClass.getClassLoader.getResource(resourceName) != null)
-      out.println(
-        Source
-          .fromResource(resourceName)
-          .getLines()
-          .mkString("\n")
-      )
+  private val classLoader = getClass.getClassLoader
+
+  private val resourceName = "banner.txt"
+
+  private def banner: String =
+    if (classLoader.getResource(resourceName) != null)
+      Source
+        .fromResource(resourceName, classLoader)
```

Comment:
super minor thing, but it feels a bit strange to get resource just for checking for `null` and then ignoring it.

how about this?:

```
  Option(classLoader.getResourceAsStream(resource)) match {
      case Some(in) => Source.fromInputStream(in).getLines().mkString("\n")
      case None     => "Banner resource missing from classpath."
    }
```

Reply:
even better :D